src/org/omegat/gui/main/ProjectUICommands.java

```diff
@@ -1315,6 +1316,40 @@ public static void doWikiImport() {
         }
     }
 
+    public static void openWritableGlossaryFile(boolean parent) {
```

Comment:
```suggestion
    public static void openWritableGlossaryFile(boolean parentDirectory) {
```

It is better argument name to avoid ambiguous against parent window.